src/tables/head.js

```diff
@@ -52,7 +52,7 @@ function makeHeadTable(options) {
         {name: 'yMin', type: 'SHORT', value: 0},
         {name: 'xMax', type: 'SHORT', value: 0},
         {name: 'yMax', type: 'SHORT', value: 0},
-        {name: 'macStyle', type: 'USHORT', value: 0},
+        {name: 'macStyle', type: 'USHORT', value: options.macStyle},
```

Comment:
might be good to add a default value `|| 0` just in case

Reply:
I resolved this by adding the line 

```javascript
let macStyle = options.macStyle || 0;
```